src/app/modules/main/chat_section/module.nim

```diff
@@ -738,7 +738,10 @@ method onCommunityChannelDeletedOrChatLeft*(self: Module, chatId: string) =
   self.view.chatsModel().removeItemById(chatId)
   self.removeSubmodule(chatId)
 
-  self.setFirstChannelAsActive()
+  # check if the currently active chat has been deleted
```

Comment:
Hi there, nice finding, the comment seems confusing to me. 
I was expecting :

```nim
let activeChatId = self.controller.getActiveChatId()
if isDeleted(activeChatId):
   # take action
   ...
```

Could you explain why do we need to setFirstChannelAsActive() if the chatId is the same as the activeChat please?
In the end we might not need the comment, the code should be self explanatory unless there's a specific decision you want to explain more here...

Reply:
Sure, updated the branch